### PR TITLE
Addition to information

### PR DESCRIPTION
**Pull request by joinimran**
As we can also configure credential guard using account protection profile in endpoint security. Added link to that article as well. 

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8711

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/identity-protection/credential-guard/credential-guard-manage.md [hunk 1]
```diff
@@ -59,6 +59,9 @@ To enforce processing of the group policy, you can run ```gpupdate /force```.
 
 3.  Click **Profiles** > **Create Profile** > **Endpoint protection** > **Windows Defender Credential Guard**.
 
+> [!TIP]
+> You can also configure Credential Guard using account protection profile in Endpoint Security. See [https://docs.microsoft.com/mem/intune/protect/endpoint-security-account-protection-profile-settings](account protection policy settings for endpoint security in Intune).
```
Comment (illfated) [inline windows/security/identity-protection/credential-guard/credential-guard-manage.md:63]: You seem to have reversed the order of URL and its link text description. Let me reorder:
```suggestion
> You can also configure Credential Guard using account protection profile in Endpoint Security. See [Account protection policy settings for endpoint security in Intune](https://docs.microsoft.com/mem/intune/protect/endpoint-security-account-protection-profile-settings).
```
Also note: I copied the verbatim page title text, including the capitalized first word "Account" .

Comment (illfated) [reply windows/security/identity-protection/credential-guard/credential-guard-manage.md:63]: (Maybe JohanFreelancer9 would like to offer an improvement to the sentence, I don't know yet.)

Comment (JohanFreelancer9) [inline windows/security/identity-protection/credential-guard/credential-guard-manage.md:63]:
```suggestion
> You can also configure Credential Guard using an account protection profile in endpoint security. See [Account protection policy settings for endpoint security in Intune](https://docs.microsoft.com/mem/intune/protect/endpoint-security-account-protection-profile-settings).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by illfated — APPROVED** [review]

**Comment by nam31** [comment]
@DulceMontemayor, PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**